modules/vmware-csi-driver-reqs.adoc

```diff
@@ -24,7 +24,7 @@
 
 To install the vSphere CSI Driver Operator, the following requirements must be met:
 
-* VMware vSphere version 6.7U3 or later
+* VMware vSphere version 7.0.1 or later
```

Comment:
Should we want to mention that VMware vSphere version 7.0 Update 1 is now deprecated, and that customers should use 7.0 update 2 or later?